docs/WritingTests.asciidoc

```diff
@@ -477,13 +477,17 @@ sub run () {
 
 By default, any worker can get any job with the matching architecture.
 
-This behavior can be changed by setting job variable `WORKER_CLASS`. Jobs with
-this variable set (typically via machines or test suites configuration) are
-assigned only to workers, which have the same variable in the configuration
-file.
+This behavior can be changed by setting the job variable `WORKER_CLASS` taking
+a comma-separated list of worker class values. The values from multiple places
+of definition are combined. Typically machines and test suite configurations
+set the worker class. Jobs with this variable set are assigned only to
+workers, which have all corresponding worker class values in their
+configuration (and-combination).
 
 For example, the following configuration ensures, that jobs with
 `WORKER_CLASS=desktop` can be assigned _only_ to worker instances 1 and 2.
```

Comment:
Job with `WORKER_CLASS=desktop` can now be assigned to workers 1, 2 and 4.

Reply:
I deleted instance 4 again and extended instance 2 to have "desktop,foo,bar" so the first sentence of explanation can stay as in before